setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 
-with open('README.rst') as read_file:
-    long_description = read_file.read()
+#with open('README.rst') as read_file:
+#    long_description = read_file.read()
```

Comment:
Ah `README.rst` wasn't part of the repo. Added it, if you rebase, you should be good.